config/questions/en.yml

```diff
@@ -1,311 +1,307 @@
+  - old are you
+
+- What:
+  - [activity, activities]:
+    - do you enjoy the most
+  - apps do you use daily
+  - do you:
+    - do:
+      - for fun
+      - on the weekend
+    - like more,:
+      - dogs or cats
+      - shower or bath
+      - tea or coffee
+    - think about:
+      - Bad Dragon
+      - cats
+      - coyotes
+      - dogs
+      - dragons
+      - '"fact" accounts'
+      - foxes
+      - lizards
+      - raccoons
+      - society
+      - surveillance
+      - the Internet
+    - usually:
+      - do when you're bored
+      - have for breakfast
+  - habit do you really find cute in a person
+  - is:
+    - "heavier: a kilogram of steel, or a kilogram of feathers"
+    - one thing you would like to become better at
+    - the:
+      - [best, worst]:
+        - fast food chain
+        - thing:
+          - about:
+              - society
+              - the Internet
+              - your:
+                - country
+                - favourite:
+                  - book
+                  - movie
+                  - series
+                - hometown
+          - ever
+          - one can do if they're bored
+        - way to prepare and eat eggs
+    - your:
+      - dream job
       - favourite:
-        - Retrospring user
-        - actor
-        - artist
-        - comedian
-        - musician
-Why:
-  do:
-    - they call it oven when you of in the cold food of out hot eat the food
-Would:
-  you:
-    rather:
-      live:
-        - in:
-          - a city or in the countryside
-          - a flat or in a house
-      lose:
-        - an arm or a leg
-      be:
-        - the best player on a horrible team or the worst player on a great team
+        - activity
+        - animal
+        - beverage
+        - board game
+        - book series
+        - console
+        - food
+        - Internet browser
+        - language
+        - name
+        - number
+        - piece of music
+        - programming language
+        - season
+        - snack
+        - sports team
+        - type of cheese
+        - video game
+        - way to eat a potato
+        - website
+      - first memory
+  - kind of animals do you have
+  - languages do you know
+  - was:
+    - the:
+      - best day of your life like
+      - [best, worst]:
+        - thing you've eaten so far
+      - last:
+          - concert you've been to
+          - song you listened to on repeat
+          - thing you:
+            - ate
+            - did
+            - have:
+              - bought
+              - done
+              - eaten
+              - looked for
+    - your favourite song:
+      - as a kid
+      - from a few weeks ago
+  - would you do if you:
+    - had:
+        - one billion euros
+        - one million euros
+        - the ability to fly
+    - were a dragon
+    - woke up with:
+      - some drawings in your face
+      - someone else next to you
+
+- Where do you:
+  - live
+  - work
```

Comment:
Could come off as kind of creepy.

Maybe a "What do you do for a living?" instead of work?

Reply:
good idea